src/platform/silabs/SiWx/ble/BLEManagerImpl.cpp

```diff
@@ -23,17 +23,17 @@
  */
 
 /* this file behaves like a config.h, comes first */
-#include <platform/internal/CHIPDeviceLayerInternal.h>
+#include <platform/internal/CHIPDeviceLayerInternal.h> // nogncheck
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
-#include "wfx_sl_ble_init.h"
+#include "sl_si91x_ble_init.h"
 #include <ble/Ble.h>
 #include <crypto/RandUtils.h>
 #include <lib/support/CodeUtils.h>
 #include <lib/support/logging/CHIPLogging.h>
-#include <platform/CommissionableDataProvider.h>
-#include <platform/DeviceInstanceInfoProvider.h>
-#include <platform/internal/BLEManager.h>
+#include <platform/CommissionableDataProvider.h> // nogncheck
+#include <platform/DeviceInstanceInfoProvider.h> // nogncheck
+#include <platform/internal/BLEManager.h>        // nogncheck
```

Comment:
Why do they need a `nogncheck` now and they didn't need one before? I think we should always have those dependencies with the source_set building BLEManagerImpl.cpp